### ISSUE TITLE
Import entry from clipboard

### ISSUE THREAD

**Issue by buhtz** [JabRef/jabref#3143]
Some websites offer bibliographic data in machinereadable format. e.g. a Bibtex oder RIS text. But there is no download link. The text is just displayed so you can copy and paste it.

JabRef should be able to read from clipboard e.g. the content of a RIS file.
```
TY  - BOOK
ED  - Kempf, Ute
ED  - Wrede, Birgitta
ID  - 2912032
SN  - ISBN 978-3-932869-19-8
TI  - Gender-Effekte. Wie Frauen die Technik von morgen gestalten
VL  - Band 19
ER  - 
```

This would prevent me to open a text editor, paste the text in there, store it as a RIS file, reopen it with jabref, ... ;)

**Comment by lenhard** [comment]
We sort of have that already. 

Try in the menu: BibTeX -> New entry from plain text -> select the type -> paste the text.

Afaik, it only works for BibTeX and the workflow could certainly be optimized, though.

**Comment by buhtz** [comment]
Mhm... I see. But it could be "better" - IMO.

1. Internaly all import formats supported by JabRef should be used by the "entry from plain text" feature, too.
2. And the access to the clipboard feature should be in the "import into new database"/"import into the current database" area. I am not sure how this could be solved in the UI because the "import" currently opens a file dialog.

**Comment by stefan-kolb** [comment]
Just as a side note: pasting entries works as long as they are in BibTeX format.

**Comment by koppor** [comment]
@buhtz Can you imagine opening a pull request for that? We have [code howtos](https://github.com/JabRef/jabref/wiki/Code-Howtos) and [guidelines for setting up a local workspace](https://github.com/JabRef/jabref/wiki/Guidelines-for-setting-up-a-local-workspace). I think, your feature request is "just" iterating over all available EntryFetchers and checking if the fetcher accepts the format.

**Comment by buhtz** [comment]
@kopper Sorry I am far away from Java and out of time ressource. Just keep this Issue open to document the enhancement. Of course I understand your ressource situation also.

**Comment by lenhard** [comment]
This is fixed by #3243
